Fix setup.py to add either lookups dir to installdir
Author: Pavel Tomanek (Inuits)
--- setup.py.orig	2025-06-18 16:34:19.958477374 +0200
+++ setup.py	2025-06-18 16:36:22.079482402 +0200
@@ -6,7 +6,7 @@
 # @Last Modified time: 2023-05-18 09:06:45
 
 import os
-from setuptools import setup
+from setuptools import setup, find_packages
 
 readme_file = 'README.md'
 requirements_file = 'requirements.txt'
@@ -56,7 +56,11 @@
     author='Theo Lemaire',
     author_email='theo.lemaire1@gmail.com',
     license='MIT',
-    packages=['PySONIC'],
+    packages=find_packages(),
+    include_package_data=True,
+    package_data={
+        'PySONIC': ['lookups/*'],
+    },
     scripts=getFiles('scripts') + getFiles('tests'),
     install_requires=get_requirements(),
     zip_safe=False
